File: Examples/Mvc.VB/Controllers/DocumentViewerController.Highlight.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

    Public Function Highlight() As ActionResult
		Dim documentViewer = New DocumentViewer() With {
			.Width = 800,
			.Height = 600,
			.Resizable = True,
			.Document = "~/App_Data/ExampleFiles/Default.doc",
			.SearchOptions = New SearchOptions With
		        {
		            .Term = "ancient mariner",
		            .MatchOptions = MatchOptions.MatchAnyWord
		        }
		}

		Return View(documentViewer)
	End Function

    End Class
End Namespace

File: Examples/AspNetCoreCS/Controllers/DocumentViewerController.Highlight.cs
using GleamTech.DocumentUltimate.AspNet.UI;
using Microsoft.AspNetCore.Mvc;

namespace GleamTech.DocumentUltimateExamples.AspNetCoreCS.Controllers
{
    public partial class DocumentViewerController
    {
        public IActionResult Highlight()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 800,
                Height = 600,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.doc",
                SearchOptions =
                {
                    Term = "ancient mariner",
                    MatchOptions = DocumentViewerMatchOptions.MatchAnyWord
                }
            };

            return View(documentViewer);
        }
    }
}
